modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -290,15 +290,17 @@ NSFVAction::validParams()
       std::vector<MooseFunctorName>(),
       "Functor names for the sources used for the passive scalar fields.");
 
-  params.addParam<CoupledName>(
+  params.addParam<std::vector<CoupledName>>(
```

Comment:
```suggestion
  params.addParam<std::vector<std::vector<VariableName>>>(
```

Reply:
@grmnptr added this alias for simplicity

Reply:
It just seemed like you had changed from `CoupledName` to `std::vector<VariableName>` in other places in this PR, so I figured you would want to be consistent

Reply:
umm I might have done that.
I ll check

Reply:
yeah I changed one. I ll change it back
I was indeed deleting it at first because CoupledName isnt intuitive. But then I reversed my changes